### PR TITLE
436 add highlight color to the header menu

### LINKED ISSUE #436: add highlight color to the header menu
add highlight color to the header menu to show which site is active

### PR DESCRIPTION
**Pull request by LetsPoke**
closes #436

### CODE REVIEW

File: hms-app/src/common/constants.ts [hunk 1]
```diff
@@ -1 +1,3 @@
 export const NULL_DATE = new Date('1970-01-01')
+export const HeaderActiveColor = 'lightblue'
```
Comment (verzano) [inline hms-app/src/common/constants.ts:2]: so I think there is already a `styles` file that this wold fit more logically in

Comment (verzano) [reply hms-app/src/common/constants.ts:2]: also the capitalization is probably wrong, constants should be ALL_CAPS or possibly camalCase

File: hms-app/src/components/HeaderMenu.tsx [hunk 2]
```diff
@@ -18,10 +19,21 @@ export default function HeaderMenu({ links }: HeaderSearchProps) {
...
+        key={link.label}
+        to={link.link}
+        className={classes.link}
+        style={{
+          backgroundColor:
+            location.pathname.slice(1) === link.link
+              ? HeaderActiveColor
+              : undefined,
+        }}
+      >
```
Comment (verzano) [inline hms-app/src/components/HeaderMenu.tsx:36]: all of this should probably be defined in the `styles` file

Comment (verzano) [reply hms-app/src/components/HeaderMenu.tsx:36]: could make a new class called... headerLink or something...

Comment (LetsPoke) [reply hms-app/src/components/HeaderMenu.tsx:36]: yeah but then you would to pass in location.pathname and link.link?

**Review by verzano — APPROVED** [review]

**Pull request merged**